roles/pulp_common/tasks/install_pip.yml

```diff
@@ -132,6 +132,11 @@
         virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
       when: prereq_pip_packages | length > 0
 
+    - name: Install a patched version of Django LTS (FIPS only)
+      pip:
+        name: git+https://github.com/mdellweg/django.git@fips
+      when: ansible_fips
+
```

Comment:
When in FIPS mode, we should also add it (django at its reported version string) to the constraints file.

So that no plugin or "pip install --upgrade" will blow it away:
https://github.com/pulp/pulp_installer/blob/5c8d9e0715ebb1173da7cb7fa2ee8db635c9fd6a/roles/pulp_common/tasks/install_pip.yml#L184-L187

https://pip.pypa.io/en/stable/user_guide/#constraints-files

Reply:
This is a good idea, +1